features/shared/checkout_previous_branch_after_success/renamed_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,11 @@
+Feature: when the current branch is renamed during a Git Town command, the previous branch history is preserved
```

Comment:
Feature: renaming the current branch maintains the existing previous branch setting